arbor/fvm_layout.cpp

```diff
@@ -337,8 +331,8 @@ fvm_cv_discretize(const cable_cell& cell, const cable_cell_parameter_set& global
         for (msize_t i = 0; i<n_branch; ++i) {
             auto cable = mcable{i, 0., 1.};
             auto scale_param = [&, ion=ion](const auto&,
-                                   const inv_diff& par) {
-                auto ie = thingify(par.value, provider);
+                                   const inv_diff& par) -> double {
+                auto ie = thingify(par.value, provider); // TODO
```

Comment:
TODO?

Reply:
That was eventually solved when constructing the inverse diffusivity above. Done.